WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -43,7 +43,8 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
 
         wordPressComRestApi.GET(path, parameters: parameters, success: { (responseObject, httpResponse) in
             guard let response = responseObject as? [String: AnyObject],
-                      people = try? self.peopleFromResponse(response, siteID: siteID, type: User.self) else
+                users = response["users"] as? [[String: AnyObject]],
+                people = try? self.peopleFromResponse(users, siteID: siteID, type: User.self) else
```

Comment:
On purpose, actually. As soon as the call wraps up, the closures get released, and the self reference cleared out